app/lib/actions/katello/upstream_subscriptions/remove_entitlement.rb

```diff
@@ -6,10 +6,13 @@ class RemoveEntitlement < Actions::Base
 
         input_format do
           param :entitlement_id
+          param :sub_name
         end
 
         def run
           output[:response] = ::Katello::Resources::Candlepin::UpstreamConsumer.remove_entitlement(input[:entitlement_id])
+        rescue ::Katello::Errors::EntitlementGone
```

Comment:
Nice. One thing, can we call it UpstreamEntitlementGone? Just to make it a little more clear since we could conceivably have an error of the same name pertaining to the local Candlepin :)

Reply:
Sure thing!